src/figuresClasses.ts

```diff
@@ -1,11 +1,73 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
`type Color = 'red' | 'green' | 'blue'`

```suggestion
  color: Color;
```
and now you can reuse the Color everywhere